core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala

```diff
@@ -57,6 +58,11 @@ class BlockManagerDecommissionIntegrationSuite extends SparkFunSuite with LocalS
       .set(config.STORAGE_DECOMMISSION_ENABLED, true)
       .set(config.STORAGE_DECOMMISSION_RDD_BLOCKS_ENABLED, persist)
       .set(config.STORAGE_DECOMMISSION_SHUFFLE_BLOCKS_ENABLED, shuffle)
+      // Force exactly one executor per worker such that all block managers
+      // get the shuffle and RDD blocks.
+      .set(config.EXECUTOR_CORES.key, "1")
+      .set(config.CPUS_PER_TASK.key, "1")
+      .set(config.EXECUTOR_MEMORY.key, "1024m")
```

Comment:
I think these changes are redundant and not needed as they are setting the same values as the defaults for these configs:

https://github.com/apache/spark/blob/a939c675dd70ec0368e7d9d8f08f1ffca662cf1d/core/src/main/scala/org/apache/spark/internal/config/package.scala#L293-L302

https://github.com/apache/spark/blob/a939c675dd70ec0368e7d9d8f08f1ffca662cf1d/core/src/main/scala/org/apache/spark/internal/config/package.scala#L524-L525